app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -441,11 +505,17 @@ private void displayCurrentInfoFromReading(BgReading lastBgReading, boolean pred
             }
         }
         int minutes = (int)(System.currentTimeMillis() - lastBgReading.timestamp) / (60 * 1000);
-        notificationText.append("\n" + minutes + ((minutes==1)?" Minute ago":" Minutes ago"));
+        String minutesString;
+        if(BgGraphBuilder.isXLargeTablet(getApplicationContext()) || BgGraphBuilder.isLargeTablet(getApplicationContext())) {
+            minutesString = " Min ago";
```

Comment:
As the extra status line now is a separate text field is this short form still needed?